### PR TITLE
jwt_verify_lib@0.0.0-20230517-b59e807

### PR DESCRIPTION
**Pull request by mmorel-35**
Used by:
* https://github.com/envoyproxy/envoy

Related to: https://github.com/google/jwt_verify_lib/pull/103

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (jwt_verify_lib) have been updated in this PR. Please review the changes.

File: modules/jwt_verify_lib/0.0.0-202230517-b59e807/source.json
Comment (fmeum) [inline modules/jwt_verify_lib/0.0.0-202230517-b59e807/source.json:2]: Verified that this is not from a fork: https://github.com/google/jwt_verify_lib/commit/b59e8075d4a4f975ba6f109e1916d6e60aeb5613

**Comment by fmeum** [comment]
Blocked on a rules_fuzzing release with https://github.com/bazelbuild/rules_fuzzing/pull/248.

File: modules/jwt_verify_lib/0.0.0-202230517-b59e807/source.json
Comment (fmeum) [reply modules/jwt_verify_lib/0.0.0-202230517-b59e807/source.json:2]: Yes, everything is fine, it's just something we need to check for this particular GitHub endpoint.

**Comment by fmeum** [comment]
The release failed because the archive is a zip, not a tar.gz. We'll have to submit it manually. I'm somewhat busy with the next Bazel release right now, so if you want it quicker, you could submit it and I can review it.

File: modules/jwt_verify_lib/0.0.0-202230517-b59e807/patches/jwt_verify_lib.patch
Comment (fmeum) [inline modules/jwt_verify_lib/0.0.0-202230517-b59e807/patches/jwt_verify_lib.patch:9]: Could you try to submit this patch upstream first? We don't have to wait for long before merging this as is, not at least we can link the PR.

**Review by fmeum — APPROVED** [review]

**Pull request merged**
